contracts/airdrops/Existing1155NftDrop.sol

```diff
@@ -7,6 +7,11 @@ import "@openzeppelin/contracts/token/ERC1155/IERC1155Receiver.sol";
 import "@openzeppelin/contracts/access/Ownable.sol";
 import "../interfaces/AirdropMerkleProof.sol";
 import "../interfaces/AirdropInfo.sol";
+// import "../interfaces/IAirBroFactory.sol";
+
+interface IAirBroFactoryExisting1155Nft {
```

Comment:
There is no need for multiple factory interfaces, you need just one IAirBroFactory interface like the one you imported above but for some reason it's commented out. This has to be updated.

Reply:
This should solved now.

Reply:
Great, code looks much cleaner now.